diff --git a/browser/chrome/browser/browser.dtd b/browser/chrome/browser/browser.dtd
--- a/browser/chrome/browser/browser.dtd
+++ b/browser/chrome/browser/browser.dtd
@@ -183,17 +183,17 @@ These should match what Safari and other
 <!ENTITY toolsMenu.accesskey          "E"> 
 
 <!ENTITY keywordfield.label           "Kulcsszó hozzáadása a kereséshez…">
 <!ENTITY keywordfield.accesskey       "u">
 <!ENTITY search.label                 "Keresés a weben">
 <!ENTITY search.accesskey             "K">
 <!ENTITY downloads.label              "Letöltések">
 <!ENTITY downloads.tooltip            "A folyamatban lévő letöltések állapotának megjelenítése">
-<!ENTITY downloads.accesskey          "e">
+<!ENTITY downloads.accesskey          "L">
 <!ENTITY downloads.commandkey         "j">
 <!ENTITY downloadsUnix.commandkey     "y">
 <!ENTITY addons.label                 "Kiegészítők">
 <!ENTITY addons.accesskey             "T">
 <!ENTITY addons.commandkey            "A">
 
 <!ENTITY webDeveloperMenu.label       "Webfejlesztő">
 <!ENTITY webDeveloperMenu.accesskey   "W">
